cmd/telemetrygen/internal/metrics/config.go

```diff
@@ -22,6 +22,9 @@ func (c *Config) Flags(fs *pflag.FlagSet) {
 	c.MetricType = metricTypeGauge
 
 	c.CommonFlags(fs)
+
+	fs.StringVar(&c.HTTPPath, "otlp-http-url-path", "/v1/metrics", "Which URL path to write to")
```

Comment:
please add a test for this parameter

Reply:
certainly, would a test checking if the exporter is created with `otlptracehttp.WithURLPath(cfg.HTTPPath)` be suffice?

Reply:
Yup! that works

Reply:
I gave it a shot but ran into an issue with most of the attributes and structs I needed for writing the test being internal to the `otlp{trace,metrics}{grpc,http}` packages (like the option functions and the config structs they're applied to).

I'm maybe missing something with the above, but the best I was able to come up with was a test I've added in `config_test.go` which just simply  checks the defaults are different per command.